src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -158,6 +158,21 @@ class FunSpecTest {
       |""".trimMargin())
   }
 
+  @Test fun functionParamWithKdoc() {
```

Comment:
Can you add a test that has a param with KDoc and a param without the KDoc?

Reply:
I added a couple of additional parameters to this test, I hope that's OK.